src/lib/Exception/ItemNotFoundException.php

```diff
@@ -12,10 +12,16 @@
 
 final class ItemNotFoundException extends NotFoundException
 {
-    public function __construct(string $itemId, string $language, int $code = 0, Throwable $previous = null)
+    public function __construct(?string $itemId = null, ?string $language = null, int $code = 0, Throwable $previous = null)
     {
+        $message = 'Item not found';
+
+        if (isset($itemId, $language)) {
+            $message .= sprintf(' with id: %s and language: %s', $itemId, $language);
+        }
```

Comment:
Since now you have `OutOfBoundsException`, these changes are no longer needed. Considering they introduced relaxed types, they should be reverted back :smile_cat: